src/sections/Community/Handbook/images/community-green.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 59.3 44.5"><defs><style>.cls-1{fill:#00b39f}.cls-2{fill:#00d3a9}</style></defs><title>Asset 1</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><path d="M45.5,16.9a13.79,13.79,0,0,0-6.4,1.6,9.52,9.52,0,0,1-4.6,7.6,16,16,0,0,1,9.2,7.7H58.2a1.11,1.11,0,0,0,1.1-1.1V30.6A13.81,13.81,0,0,0,45.5,16.9Z" class="cls-1"/><path d="M13.8,16.9A13.83,13.83,0,0,0,0,30.7v2.1a1.11,1.11,0,0,0,1.1,1.1H15.7a16,16,0,0,1,9.2-7.7,9.55,9.55,0,0,1-4.7-7.7,14.11,14.11,0,0,0-6.4-1.6Z" class="cls-1"/><path d="M29.6,10.6A7.4,7.4,0,1,0,37,18,7.38,7.38,0,0,0,29.6,10.6Z" class="cls-2"/><path d="M29.6,27.5A13.83,13.83,0,0,0,15.8,41.3v2.1a1.11,1.11,0,0,0,1.1,1.1H42.3a1.11,1.11,0,0,0,1.1-1.1h0V41.3A13.76,13.76,0,0,0,29.6,27.5Z" class="cls-2"/><path d="M45.5,0a7.4,7.4,0,1,0,7.4,7.4A7.38,7.38,0,0,0,45.5,0Z" class="cls-1"/><path d="M13.8,0a7.4,7.4,0,1,0,7.4,7.4A7.38,7.38,0,0,0,13.8,0Z" class="cls-1"/></g></g></svg>
```

Comment:
Most of these logos are already available in the repository under `assets/images` folder. Can you import it from there itself instead of creating new files here.